test/test-index.js

```diff
@@ -197,3 +220,73 @@ test.cb('The task receives the cumulus_config property', (t) => {
   return cumulusMessageAdapter
     .runCumulusTask(businessLogic, inputEvent, context, t.end);
 });
+
+test('GetMessageGranules returns empty array if no granules are found', (t) => {
+  const messageGranules = getMessageGranules(testContext.inputEvent);
+
+  t.deepEqual(messageGranules, []);
+});
+
+test('GetMessageGranules returns granules if they are in the payload', (t) => {
+  const messageGranules = getMessageGranules(testContext.granuleInputJson);
+
+  t.deepEqual(messageGranules, [
+    'MOD09GQ.A2016358.h13v04.006.2016360104606',
+    'MOD09GQ.A2016358.h13v04.007.2017'
+  ]);
+});
+
+test('GetMessageGranules returns granules if they are in the meta.input_granules', (t) => {
```

Comment:
I think this is OBE by nasa/cumulus#1313  https://github.com/nasa/cumulus/pull/1313/files#diff-2fdbb43ab43a3b0596451c3fa38ecbc9R373